src/main/java/io/github/thebusybiscuit/slimefun4/core/commands/SlimefunTabCompleter.java

```diff
@@ -30,6 +31,7 @@ public SlimefunTabCompleter(@Nonnull SlimefunCommand command) {
 
     @Nullable
     @Override
+    @ParametersAreNonnullByDefault
     public List<String> onTabComplete(CommandSender sender, Command cmd, String label, String[] args) {
```

Comment:
```suggestion
    @Override
    @ParametersAreNonnullByDefault
    public @Nullable List<String> onTabComplete(CommandSender sender, Command cmd, String label, String[] args) {
```
You also missed some other return type annotations in this file, but github won't let me comment on them since you didn't change them.